Loadouts/msv_csat_sama_hex.hpp

```diff
@@ -0,0 +1,280 @@
+};
+class potato_msv_mmgg: potato_msv_AR {// MMG
+  backpack[] = {"B_FieldPack_ocamo"};
+  weapons[] = {MMG};
+  magazines[] = {MMG_MAG,PISTOL_MAG,BASE_GRENADES};
+};
+class potato_msv_mmgag: Fic_Spotter {// MMG Spotter/Ammo Bearer
+  backpack[] = {"B_FieldPack_ocamo"};
+  backpackItems[] += {MMG_MAG};
+};
+class potato_msv_matg: potato_msv_rifleman {// MAT Gunner
+  backpack[] = {"B_Carryall_ocamo"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG};
+  items[] += {BASE_MEDICAL};
+  launchers[] = {MAT};
+};
+class potato_msv_matag: Fic_Spotter {// MAT Spotter/Ammo Bearer
+  backpack[] = {"B_Carryall_ocamo"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG2};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_msamg: potato_msv_rifleman {// SAM Gunner
+  SAM_GEAR("B_Carryall_ocamo", SAM_MAG)
+  launchers[] = {SAM};
+};
+class potato_msv_msamag: Fic_Spotter {// SAM Spotter/Ammo Bearer
+  SAM_GEAR("B_Carryall_ocamo", SAM_MAG2)
+};
+class potato_msv_msaml: potato_msv_msamag {// SAM Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RADIO_MR};
+};
+class potato_msv_mtrl: Fic_Spotter { // Mortar Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER,RADIO_LR};
+};
+class potato_msv_mtrg: potato_msv_rifleman {// Mortar Gunner
+  MORTAR_GEAR("O_Mortar_01_weapon_F")
+};
+class potato_msv_mtrag: Fic_Spotter {// Assistant Mortar
+  MORTAR_GEAR("O_Mortar_01_support_F")
+};
+class potato_msv_pilot {// Pilot
+  uniform[] = {"U_O_PilotCoveralls"};
+  backpack[] = {"B_FieldPack_ocamo"};
+  vest[] = {"V_TacVest_khk"};
+  headgear[] = {"H_PilotHelmetHeli_O"};
+  weapons[] = {SMG};
+  magazines[] = {SMG_MAG,CREW_GRENADES};
+  backpackItems[] = {RADIO_LR};
+  items[] = {BASE_MEDICAL,BASE_TOOLS,LEADER_TOOLS,RADIO_MR};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_cc: potato_msv_pilot {// Crew Chief
+  headgear[] = {"H_CrewHelmetHeli_O"};
+};
+class fic_vehicle_crewman: potato_msv_rifleman {// Vehicle crew base
+  headgear[] = {"H_HelmetCrew_O"};
+  magazines[] = {RIFLE_MAG,CREW_GRENADES};
+  backpackItems[] = {KEY};
+  linkedItems[] += {BASE_LINKED,LEADER_LINKED};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_vicc: fic_vehicle_crewman {// Vehicle Crewman (non-repair)
+  backpackItems[] += {RADIO_MR};
+  linkedItems[] += {BINOS};
+};
+class potato_msv_vicl: potato_msv_vicc {// Vehicle Leader
+  backpackItems[] += {RADIO_LR};
+};
+class potato_msv_vicd: fic_vehicle_crewman {// Vehicle Crewman (repair)
+  backpackItems[] += {"toolkit"};
+};
+class potato_msv_eng: potato_msv_rifleman {// Demoman
+  backpack[] = {"B_Carryall_ocamo"};
+  items[] += {BASE_ENG,BASE_MEDICAL};
+  backpackItems[] = {"Toolkit"};
+  magazines[] += {MSV_EXP};
+  linkedItems[] = {BASE_LINKED};
+};
+class potato_msv_engl: potato_msv_eng {// Demoman Leader
+  backpackItems[] += {RADIO_MR,KEY};
+  linkedItems[] += {LEADER_LINKED};
+};
+class potato_msv_sf_rifleman: potato_msv_rifleman {// Recon Rifleman
```

Comment:
It would technically be possible to use the other VIPER camo variant for this recon team, as there is a normal CSAT hex for them, but it's not really important.

Reply:
All viper gear is APEX (which would defeat the purpose of this loadout)

Reply:
APEX is required though, so using its content was okay I thought.

Reply:
The point to most of these is to be as straight vanilla as possible. I added APEX specific loadouts in addition to the vanilla loadouts, but not as augmentations/in lieu of